src/pages/[[...markdownPath]].js

```diff
@@ -216,10 +217,14 @@ export async function getStaticProps(context) {
   const fm = require('gray-matter');
   const meta = fm(mdx).data;
 
+  // Serialize MDX into JSON.
```

Comment:
Can probably revert this now.

Reply:
Reverted in [`90586e4` (#6214)](https://github.com/reactjs/react.dev/pull/6214/commits/90586e42ad98c72de3c285087ca231c51bdfa62d).